src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1

```diff
@@ -856,3 +856,66 @@ function Test-VirtualMachineScaleSetIdentity
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Test Virtual Machine Scale Set Networking part
+#>
+function Test-VirtualMachineScaleSetNetworking
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName
+    $ipName = Get-ComputeTestResourceName
+    $nsgName = Get-ComputeTestResourceName
+
+    try
+    {
+        # Common
+        $loc = 'southeastasia';
```

Comment:
Please do not hardcode the location.  This prevents this test from being used with subscriptiosn that do not have access there, or in environments that do not have this location.  Instead, query ARM for the available locations for the providers thaty are needed for this test.  You *can* prefer a location if it appears in the list.